content/en/docs/concepts/services-networking/cluster-ip-allocation.md

```diff
@@ -0,0 +1,139 @@
+---
+reviewers:
+- sftim
+- thockin
+title: Service ClusterIP allocation
```

Comment:
Is it a `ClusterIP`, or just a virtual cluster IP? I'm not sure on the need for writing this text monospace. Bear in mind that the Service that gets one of these virtual IP addresses might not have the `type` field set to `"ClusterIP"`. For example, it could be a Service of `type: LoadBalancer`.

Reply:
this is ClusterIP specific, the LoadBalancer IP is assigned by an external controller and not explained here ... all Services type have a ClusterIP except, headless that use `ClusterIP: None` or those with type ExternalName

Reply:
> all Services type have a ClusterIP 

Right, that's the bit where I'm worried we're confusing readers. All services have a thing, but should we call the thing a `ClusterIP`? To me `ClusterIP` is a particular enum value. We wouldn't, for example, write:
“the packets are rewritten as they pass through the `LoadBalancer`”, even though `LoadBalancer` is the type of Service involved there.